package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "app-management-app",
-    "version": "31.0.0",
+    "version": "31.2.0",
```

Comment:
Any reason for two minor bumps?

Reply:
Just pushed twice to the App Hub... though I deleted the older versions.  We're going to want to have some more sane versioning at some point soon so likely not a big deal, but I can change it to 31.1.0 if you want